CHANGE.md

```diff
@@ -0,0 +1,60 @@
+# Critical Updates for POD.jl
+
+I've forked the master branch hosted by lanl-ansi to my own Github while keeping all the most recent features merged on my master. Current, I am testing MPC instances with my master branch. Here is a summary of what are changed:
+
+## Merged Features
+* `disc_ratio` selection algorithm
+* logarithm embedding formulation
+* bounding model warm starting and no-good-cuts using solution pool
+* `disc_vars` selection algorithm based on weighted minimum vertex cover
+* support for binary/integer problem
+* support for `sin` and `cos` functions
+* rounding heuristic and multi-start heuristic for feasible solutions
+* recursive linear lifting
+* some attempts to reason bounds for variables with infinite bounds
+
+## New Operators
+* `BINPROD`: product of binary vars
+* `BINLIN`: product of a binary var and continuous var
+* `BININT`: product of a binary var and integer var
```

Comment:
are these just parsing operations or do you have some kind of relaxation?

Reply:
Updated